### ISSUE TITLE
Crude Burner only produces as much RPM as a regular furnace despite using the Blast Furnace in its recipe

### ISSUE THREAD

**Issue by orchidalloy** [mrh0/createaddition#76]
It's 16k SU while a blast furnace produces 32k SU. Given that the fluid burner is more "advanced" it makes sense to me that it would be faster than a regular furnace just as the blast furnace.

**Comment by orchidalloy** [comment]
On inspection it seems this behavior is hardcoded in the Create mod: https://github.com/Creators-of-Create/Create/blob/12bfc78443010dc25910e9f286431bd660f59e30/src/main/java/com/simibubi/create/content/contraptions/components/flywheel/engine/FurnaceEngineTileEntity.java#L28

One could work around this by injecting code at the start of the refreshWheelSpeed function: https://github.com/Creators-of-Create/Create/blob/12bfc78443010dc25910e9f286431bd660f59e30/src/main/java/com/simibubi/create/content/contraptions/components/flywheel/engine/EngineTileEntity.java#L93

**Comment by mrh0** [comment]
Yep, might be able to get Create to fix it at their end, replacing it with a block tag or something.
